src/runtime_src/core/pcie/user_aws/shim.cpp

```diff
@@ -930,6 +931,51 @@ namespace awsbwhal {
         return size;
     }
     
+    /*
```

Comment:
This code is common between linux shim, aws shim, edge shim (probably), and windows shim.  The common part should be moved to src/runtime_src/core/common/message,h,cpp.

Reply:
It is already using xrt_core::message::send common code.